spark/src/main/scala/org/apache/iceberg/spark/SparkTableUtil.scala

```diff
@@ -516,6 +413,9 @@ object SparkTableUtil {
       stagingDir: String): Unit = {
 
     implicit val manifestFileEncoder: Encoder[ManifestFile] = Encoders.javaSerialization[ManifestFile]
+    implicit val dataFileEncoder: Encoder[DataFile] = Encoders.javaSerialization[DataFile]
+    implicit val pathDataFileEncoder: Encoder[(String, DataFile)] = Encoders.tuple(Encoders.STRING,
```

Comment:
nit: this can fit into 1 line if we reuse `dataFileEncoder`:

```
implicit val pathDataFileEncoder: Encoder[(String, DataFile)] = Encoders.tuple(Encoders.STRING, dataFileEncoder)
```

Reply:
Oh, my bad.